webpack/scenes/AlternateContentSources/ACSActions.js

```diff
@@ -78,6 +80,30 @@ export const refreshACS = (acsId, handleSuccess) => post({
   errorToast: error => acsErrorToast(error),
 });
 
+export const bulkRefreshACS = (params, handleSuccess) => post({
+  type: API_OPERATIONS.POST,
+  key: BULK_ACS_REFRESH_KEY,
+  url: api.getApiUrl('/alternate_content_sources/bulk/refresh'),
+  params,
+  handleSuccess: (response) => {
+    renderTaskStartedToast(response?.data);
```

Comment:
Since this renderTaskStartedToast is contained inside bulkRefreshACS , we should probably try to get it to say that Bulk refresh started or Bulk delete started.  renderTaskStartedToast will probably need to be updated a little. Feel free to tackle it here or open an issue for this so we can handle and test this on other bulk task toasts.

Reply:
I added an override string to renderTaskStartedToast